Xabe.AutoUpdater.Test/FtpProviderTest.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Xabe.AutoUpdater.Test
+{
+    [TestClass]
+    public class FtpProviderTest
```

Comment:
This test doesn't test anything.
It's nice you added tests but without any assertion it aren'h helpfull.
Best option I would suggest it's to check if all events occured and only then pass test as correct.

Reply:
effectivelly for now, it's not a test, it's more for testing easily a provider. i wil add feature on the test.